inference-engine/src/vpu/graph_transformer/src/stages/split.cpp

```diff
@@ -153,18 +153,57 @@ Stage StageBuilder::addSplitStage(
         const DataVector& outputs) {
     std::vector<DimValues> offsets;
     offsets.reserve(outputs.size());
-
     DimValues curOffset({{axis, 0}});
-    for (const auto& output : outputs) {
-        offsets.emplace_back(curOffset);
-        curOffset.set(axis, curOffset[axis] + output->desc().dim(axis));
-    }
 
-    auto stage = addSplitStage(model, name, layer, std::move(offsets), input, outputs);
+    const auto haveUnusedOutput = [](const DataVector& outputs) {
+        auto predicate = [](const vpu::Data& out) {
```

Comment:
Nitpick: no need to define it separately, just create it in `any_of` function call.

Reply:
Changed.